bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/dto/HeatingModel.java

```diff
@@ -222,8 +222,8 @@ public String toString() {
 
     /**
      * Converts a celsius value to a FRITZ!Box value.
-     * Valid celsius values: 8 to 28 °C > 16 to 56
-     * 16 <= 8°C, 17 = 8.5°C...... 56 >= 28°C, 254 = ON, 253 = OFF
+     * Valid celsius values: 8 to 28 °C > 16 to 56,
+     * 16 &lt;= 8°C, 17 = 8.5°C...... 56 >= 28°C, 254 = ON, 253 = OFF
```

Comment:
Just curiosity why should < be changed to html antities and  > not ?

Reply:
< is the open tag marker so html will look for a known tag after this character.... no need to escape >